extensions/amp-story/0.1/amp-story.js

```diff
@@ -1092,6 +1100,18 @@ export class AmpStory extends AMP.BaseElement {
   }
 
 
+  /**
+   * Preloads the bookend config if on the last page.
+   * @private
+   */
+  maybePreloadBookend_() {
+    const pageIndex = this.getPageIndex(this.activePage_);
+    if (pageIndex + 1 >= this.getPageCount()) {
+      this.loadBookendConfig_();
```

Comment:
I think we may as well call `this.buildBookend_()` here to actually pre-render it as well.

Reply:
Done